tests/e2e/specs/in-game/gameOver.spec.js

```diff
@@ -401,6 +401,50 @@ describe('Stalemates', () => {
   });
 });
 
+describe('Conceding while a oneOff is being resolved - prevents resolving oneOff state from persisting to new game', () => {
+  beforeEach(() => {
+    cy.setupGameAsP0();
+  });
+
+  it('Opponent concedes while seven oneOff is being resolved', () => {
+    cy.loadGameFixture({
+      p0Hand: [Card.SEVEN_OF_CLUBS],
+      p0Points: [Card.SEVEN_OF_DIAMONDS, Card.SEVEN_OF_HEARTS],
+      p0FaceCards: [],
+      p1Hand: [],
+      p1Points: [],
+      p1FaceCards: [],
+      topCard: Card.FOUR_OF_CLUBS,
+      secondCard: Card.SIX_OF_DIAMONDS,
+    });
+    cy.get('[data-player-hand-card]').should('have.length', 1);
+    cy.log('Fixture loaded');
+
+    cy.playOneOffAndResolveAsPlayer(Card.SEVEN_OF_CLUBS);
+
+    cy.get('[data-top-card=4-0]').should('exist').and('be.visible');
+    cy.get('[data-second-card=6-1]').should('exist').and('be.visible');
+
+    cy.concedeOpponent();
+    assertVictory();
+    goHomeJoinNewGame();
+
+    cy.loadGameFixture({
+      p0Hand: [Card.SEVEN_OF_CLUBS],
+      p0Points: [],
+      p0FaceCards: [],
+      p1Hand: [],
+      p1Points: [],
+      p1FaceCards: [],
+      topCard: Card.FOUR_OF_CLUBS,
+      secondCard: Card.SIX_OF_DIAMONDS,
+    });
+
+    cy.get('[data-top-card=4-0]').should('not.exist');
+    cy.get('[data-second-card=6-1]').should('not.exist');
+  });
+});
```

Comment:
Nice test case. Clear and a solid replication. I'm wondering if we could add two additional test cases -- one for your opponent conceding while you resolve a four, and another for the player (not opponent) conceding during a three

Reply:
I suspect all of these cases would fail without the implemented fix as they share the same root cause of not resetting state before starting a new game